src/Directory.build.props

```diff
@@ -34,6 +34,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(IsTestProject)' != 'true' and '$(SourceLinkEnabled)' != 'false'">
-    <PackageReference Include="SourceLink.Create.CommandLine" Version="2.8.0" PrivateAssets="All" /> 
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.1" />
```

Comment:
Somewhat surprised we need the extra dependency, but 👍

Reply:
I copied pasted Rx to be honest. Will try without and see if it works still.